projects/netgrif-components-core/src/lib/view/case-view/components/default-case-list/abstract-default-case-list.ts

```diff
@@ -0,0 +1,66 @@
+import {EventEmitter, Inject, Input, OnDestroy, Optional, Output} from '@angular/core';
+import {Observable, Subject} from 'rxjs';
+import {Case} from '../../../../resources/interface/case';
+import {HeaderColumn} from '../../../../header/models/header-column';
+import {CaseViewService} from '../../service/case-view-service';
+import {LoggerService} from '../../../../logger/services/logger.service';
+import {NAE_TAB_DATA} from '../../../../tabs/tab-data-injection-token/tab-data-injection-token';
+import {InjectedTabData} from '../../../../tabs/interfaces';
+import {ActivatedRoute} from '@angular/router';
+import {filter, takeUntil} from 'rxjs/operators';
+import {TabbedVirtualScrollComponent} from '../../../../panel/abstract/tabbed-virtual-scroll.component';
+
+export abstract class AbstractDefaultCaseList extends TabbedVirtualScrollComponent implements OnDestroy {
```

Comment:
Should all Task/Case List components inherit from TabbedVirtualScrollComponent? Should this not be specific for virtual scrolls only?

Reply:
it is only possible for lists to inherit this component, because that component is same for task and cases